app/services/permissions_service.rb

```diff
@@ -43,6 +46,7 @@ def guest_permissions
       return true if controller == "skills" && action == "show"
       return true if controller == "professionals" && action == "index"
       return true if controller == "sessions" && action.in?(%w(new create))
+      # a registered user should not be authorized for line 50
       return true if controller == "users" && action.in?(%w(new create))
```

Comment:
oh i see where you coming from @AliSchlereth, because a requester for example, might also want to create a professional account, right?  We should definitely discuss permissions because we are building so much functionality dependent upon  them.